lib/application.ex

```diff
@@ -3,19 +3,19 @@ require Logger
 defmodule BlueHarvest.Application do
   use Application
 
-  @port 9000
-
   def start(_type, _args) do
+    port = Integer.parse(System.get_env("PORT")) || 4000
```

Comment:
I think you forgot to push your latest commit which fixes this to `String.to_integer/1`